andela/elrond-rc
# Ft/user onboarding/151208539

#### What does this PR do?
It onboards an admins or vendors to the application immediately they signed up and they can also look at the guides any time they wish
#### Description of Task to be completed?
As a user (vendor) who just signed up , I want to quickly see actions I can perform such that when I finished performing these actions, I will be in a good state to publish products or perform any other operations available to vendors.
#### How should this be manually tested?
Click the admin dropdown and from the options provided, click the "Get Started" and you will be routed to the page to see the guides.
#### Any background context you want to provide?
This feature has e2e implemented demonstrating the functionality.
#### What are the relevant pivotal tracker stories?
#151208539
#### Screenshots (if appropriate)
<img width="1280" alt="onboarding" src="https://user-images.githubusercontent.com/20380537/31308373-e9bc8806-ab6d-11e7-929a-03aa88028746.png">

#### Questions:
N/A
